getting-started/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting started with Quarkus
+
+This is a minimal CRUD service exposing a couple of endpoints over REST.
+
+Under the hood, this demo uses:
+
+- RESTEasy to expose the REST endpoints
+- REST-assured and JUnit 5 for endpoint testing
+
+## Requirements
+
+To compile and run this demo you will need:
+
+- JDK 1.8+
+- GraalVM
+- Apache Maven `3.5.3+`
+
+### Configurating GraalVM and JDK 1.8+
+
+1. Install GraalVM if you haven't already. You have a few options for this:
+     - Use platform-specific install tools like [homebrew](https://github.com/graalvm/homebrew-tap) or
+       [sdkman](https://sdkman.io/jdks#Oracle).
+     - Download the appropriate GraalVM archive from <https://github.com/oracle/graal/releases>,
+       and unpack it like you would any other JDK.
+2. For compile-to-native support, set the `GRAALVM_HOME` environment variable to the path of the directory containing the `bin` dir, e.g. `/opt/graalvm` on Linux or `$location/JDK/GraalVM/Contents/Home` on macOS.
+3. (Optional) To use GraalVM to run the Gradle or Maven build, set `JAVA_HOME` to the same value as `GRAALVM_HOME` , and add it to your path, for example:
+
+    > `export PATH=${JAVA_HOME}/bin:${PATH}`.
+
+See [our Building native image guide](https://quarkus.io/guides/building-native-image-guide)
+for more information.
+
+## Building the application
+
+Launch the Maven build on the checked out sources of this demo:
+
+> mvn package
+
+### Run Quarkus in developer mode
+
+To run the application in interactive mode (developer mode):
+
+> mvn compile quarkus:dev
+
+In this mode, changes you make to the code are immediately applied.
+Refresh your browser to redrive requests against the recompiled binaries.
+
+### Run Quarkus in JVM mode
+
+When you're done iterating in developer mode, you can run the application as a conventional jar file. First compile it:
+
+> mvn package
+
+Then run it:
+
+> java -jar ./target/using-spring-data-jpa-1.0-SNAPSHOT-runner.jar
+
+Have a look at how fast it boots, or measure the total native memory consumption.
+
+### Run Quarkus as a native application
+
+You can also create a native binary from this application without making any source code changes. A native binary removes the dependency on the JVM: everything needed to run the application on the target platform is included in the binary, allowing the application to run with minimal resource overhead.
```

Comment:
we usually call it a `native executable` rather than binary (same remark for the other occurrences).